test/tests/tests.vcxproj

```diff
@@ -77,4 +78,4 @@
     <ClCompile Include="TileElements.cpp" />
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Unrelated change.

Reply:
I have no idea what is going on here. I think Visual Studio added a new line, but doesn't show me. I'll take a look